core/lib/zksync_core/src/api_server/web3/state.rs

```diff
@@ -577,17 +622,18 @@ impl Filters {
     }
 
     /// Retrieves filter from the state.
-    pub fn get(&self, index: U256) -> Option<&TypedFilter> {
-        self.state.get(&index)
+    pub fn get_and_update_stats(&mut self, index: U256) -> Option<&TypedFilter> {
+        let installed_filter = self.state.get_mut(&index)?;
+
+        installed_filter.update_stats();
+
+        Some(&installed_filter.filter)
```

Comment:
Bikeshedding: IMO, inserting an empty line after each code line like here can harm readability. Of course, it's very hard (impossible?) to quantify the empty line rules, so feel free to ignore this comment. (IMO, an "empty line per ~5–10 statements" is a good ratio, but it's a personal preference.)

Reply:
I am usually more preferring to divide logical blocks by empty lines or in places where it becomes hard to understand what is going on (in this place it is more about logical blocks)